backend/main_original.py

```
@@ -0,0 +1,468 @@
+@app.post("/get-firebase-token")
+async def get_firebase_token(request: Request):
+    # Get Clerk user ID from frontend
+    auth_header = request.headers.get("Authorization")  # "Bearer <user_id>"
+    if not auth_header:
+        raise HTTPException(status_code=401, detail="Missing Authorization header")
+
+    # Extract user ID from Bearer token
+    try:
+        clerk_user_id = auth_header.replace("Bearer ", "").strip()
+        if not clerk_user_id:
+            raise HTTPException(status_code=401, detail="Empty user ID")
```

Ruff (0.13.1)

Abstract `raise` to an inner function

(TRY301)